question28/loginFailure.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link href="./css/style.css" rel="stylesheet" />
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <div id="js-wrapper" class="wrapper">
+            <h1>Login Failure</h1>
+        </div>
```

Comment:
IMO
It would be better to be able to go to the login or registration page after an unsuccessful login attempt.
It's not in the specifications, so I'll leave the modifications to you.

![スクリーンショット 2022-11-15 8 03 21](https://user-images.githubusercontent.com/76232904/201786403-dad0c0f3-9b8e-4d4f-8a54-40912034d191.png)

Reply:
Thank you for your review!
I thought it would be better to display "Please try again" and redirect to the login page when login fails, so I set that up. Please confirm correction.

fixed commit : https://github.com/csakyo/homework/pull/42/commits/dd632776dc6340873280c1607b86182feaa61d6f